pennylane/devices/default_qubit.py

```diff
@@ -99,6 +99,7 @@ class DefaultQubit(QubitDevice):
         "Hadamard",
         "S",
         "T",
+        "SX",
```

Comment:
I believe this would also be doable in `DefaultMixed`?

Reply:
done.  Thanks for the suggestion.